package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@azure/functions": "3.0.0",
-    "@zero-tech/data-store-core": "0.1.6",
+    "@zero-tech/data-store-core": "0.1.11",
```

Comment:
Need to be on 0.1.13, I recommend a quick PR in dev to upgrade this, then reopen this PR.

Reply:
Should be 0.1.13 now